tools/apidiff/go.mod

```diff
@@ -8,7 +8,4 @@ require (
 	github.com/spf13/cobra v1.1.3
 )
 
-replace (
-	github.com/Azure/azure-sdk-for-go => github.com/ArcturusZhang/azure-sdk-for-go v54.2.1+incompatible
-	github.com/Azure/azure-sdk-for-go/tools/internal => ../internal
-)
+replace github.com/Azure/azure-sdk-for-go/tools/internal => ../internal
```

Comment:
This will need to be removed.

Reply:
I planned to remove this in the next PR, but removing this in this PR is also fine